.github/ISSUE_TEMPLATE/user-story-issue.md

```diff
@@ -0,0 +1,23 @@
+---
+name: User Story
+about: This template provides a basic structure for user story issues.
+labels: Needs Triage
+---
+
+# Description
+As a ..., I want to ..., so I can ...
+
+<Link to Design Document or User Story>
+
+# Acceptance criteria
+
+- [ ] This is something that can be verified to show that this user story is satisfied.
+- [ ] Unit Tests are completed and code passes tests
+
+# Sprint Ready Checklist 
+- [ ] Acceptance criteria defined 
+- [ ] Team understands acceptance criteria 
+- [ ] Team has defined solution / steps to satisfy acceptance criteria 
+- [ ] Acceptance criteria is verifiable / testable 
+- [ ] External / 3rd Party dependencies identified
+- [ ] Is the user story small enough to be implemented in a short amount of time, but large enough to provide value?
```

Comment:
I love the intent of this.  Consider highlighting the value part more than the time component. EG

`Is the story sized to provide user value, but no larger? Consider if it could be broken up into two stories where both provide individual user value.`

Reply:
This was taken directly from the CSE Playbook and I think the wording would make individuals think about the size of the story without being too prescriptive.  Maybe the story is too large because the work is not needed.